StRoot/StEvent/StETofHeader.h

```diff
@@ -109,6 +111,7 @@ class StETofHeader : public StObject {
 
     std::vector< Bool_t > mMissMatchFlagVec;
     std::vector< Bool_t > mGoodEventFlagVec; 
+    std::vector< Bool_t > mHasPulsersVec; 
 
     ClassDef( StETofHeader, 3 )
```

Comment:
Addition of mHasPulserVec on line 114 requires incrementing the argument to ClassDef.